uw-frame-components/js/frame-config.js

```diff
@@ -32,6 +32,7 @@ define(['angular'], function(angular) {
             'crestalt': 'UW Crest',
             'group': 'UW-Madison',
             'mascotImg': 'img/bucky.gif',
+            'profileUrl': '/profile/emergencyInfo',
```

Comment:
I can see the reasons for making the profile URL configurable per theme. Seeing this, though, it motivates making `/profile` work for all users regardless of role, affiliations, and campus, so that the profile path can simply be `/profile` for everyone.